common/src/main/java/com/wynntils/core/features/Feature.java

```diff
@@ -18,9 +18,14 @@
  */
 public abstract class Feature {
     private ImmutableList<Condition> conditions;
+    private final String name;
 
     protected boolean enabled = false;
 
+    protected Feature(String name) {
```

Comment:
I don't really like this. :( Having an argument-less constructor for the Feature made it so much nicer. 

I think it would be better to have an abstract getName() method that the Feature subclasses implement. There seems to be no compelling reason to put this literal string in the constructor,

Reply:
Ill do that then